From f85b156304b030a4a366a01e6fce4ebf8cd96350 Mon Sep 17 00:00:00 2001
From: Andreas Schmidt <andreas.schmidt@wago.com>
Date: Fri, 25 Jan 2019 09:47:18 +0100
Subject: [PATCH 295/296] dts: setupfw: mv88e6321: set port 0 and 1 to fixed
 link

8215 and 8211 have both mv88e6321 but different phys
(sfp on 8211) on ports 0 and 1. To keeps setupfw dts
common for both devices that patch set the ports to
fixed link.

Signed-off-by: Andreas Schmidt <andreas.schmidt@wago.com>
---
 arch/arm/boot/dts/am335x-pfc200v3-setupfw.dts | 46 +++++----------------------
 1 file changed, 8 insertions(+), 38 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-pfc200v3-setupfw.dts b/arch/arm/boot/dts/am335x-pfc200v3-setupfw.dts
index 772df08..780e29c 100644
--- a/arch/arm/boot/dts/am335x-pfc200v3-setupfw.dts
+++ b/arch/arm/boot/dts/am335x-pfc200v3-setupfw.dts
@@ -105,44 +105,10 @@
 		interrupt-controller;
 		#interrupt-cells = <2>;
 
-		eeprom = /bits/ 8 <0x20 0x80 0x09 0x5E 0x40 0x80 0x09 0x5E 0x01 0x7D 0x01 0x7D 0x01 0x7D 0x01 0x7D
-				   0xF9 0x7F 0x40 0x19 0xF8 0x7F 0x60 0x94 0xF9 0x7F 0x40 0x19 0xF8 0x7F 0x80 0x94
-				   0xF9 0x7F 0x40 0x19 0xF8 0x7F 0x00 0x94 0xF9 0x7F 0x40 0x19 0xF8 0x7F 0x20 0x94
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF
-				   0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF 0xFF>;
-
-
 		mdio {
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			ext_phy0: ethernet-phy@0 {
-				compatible = "ethernet-phy-idD565.A401", "ethernet-phy-ieee802.3-c22";
-				reg = <0>;
-				max-speed = <100>;
-				interrupt-parent = <&gpio0>;
-				interrupts = <31 IRQ_TYPE_EDGE_FALLING>;
-			};
-
-			ext_phy1: ethernet-phy@1 {
-				compatible = "ethernet-phy-idD565.A401", "ethernet-phy-ieee802.3-c22";
-				reg = <1>;
-				max-speed = <100>;
-				interrupt-parent = <&gpio3>;
-				interrupts = <2 IRQ_TYPE_EDGE_FALLING>;
-			};
-
 			int_phy3: ethernet-phy@3 {
 				compatible = "ethernet-phy-id0141.0C00", "ethernet-phy-ieee802.3-c22";
 				reg = <3>;
@@ -167,17 +133,21 @@
 			port@0 {
 				reg = <0>;
 				label = "ethX1";
-				phy-handle = <&ext_phy0>;
 				phy-mode = "sgmii";
-                                phy-pwrdown;
+				fixed-link {
+					speed = <100>;
+					full-duplex;
+				};
 			};
 
 			port@1 {
 				reg = <1>;
 				label = "ethX2";
-				phy-handle = <&ext_phy1>;
 				phy-mode = "sgmii";
-                                phy-pwrdown;
+				fixed-link {
+					speed = <100>;
+					full-duplex;
+				};
 			};
 
 			/*port@2 {
-- 
2.7.4

